Random spelling updates & Vale config suggestions

liamlaverty opened the pull request:
## Description

1) updates the `Acronyms.yml` file to recognise domain specific acronyms. This should reduce the number of Vale related errors substantially (from around ~800 to ~70).


2) Typo adjustments in a number of differnet files:

- Authenticaion to Authentication
- adresses to addresses
- exising to existing
- intalled to installed
- Afer to After
- seif to serif
- Availabe to Available
- interefere to interfere
- accesss to access
- Schedualed to Scheduled
- alongwith to alongside
- Remeber to Remember
- seperator to separator
- selct to select
- cusom to custom
- Documen to Document
- Ukranian to Ukrainian
- avaibable to available

3) adjusts some files to correct their Vale errors

## Type of suggestion

* [ ✅ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ✅ ] Other

Other: Vale settings changes

## Product & version (if relevant)
V12, V11, V10


## Deadline (if relevant)

_When should the content be published?_

alina-tincas commented on 10/umbraco-cms/reference/using-ioc.md:
```diff
@@ -7,7 +7,7 @@ description: "Inversion of Control/Dependency Injection in Umbraco"
 # Inversion of Control / Dependency injection
 
-Umbraco 9 supports dependency injection out of the box. Umbraco uses the [ASP.NET Core built-in dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#service-lifetimes), this means that you don't have to install an external package to register and use your dependencies, and if you're familiar with APS.NET Core, the experience will ...
+Umbraco 9 supports dependency injection out of the box. Umbraco uses the [ASP.NET Core built-in dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#service-lifetimes). This means that you don't have to install external packages to register and use your dependencies. If you're familiar with ASP.NET Core, the experience will be si ...
```
It is not only in v9 but v9+ that this is supported 🙈 
```suggestion
Umbraco v9+ supports dependency injection out of the box. Umbraco uses the [ASP.NET Core built-in dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#service-lifetimes). This means that you don't have to install external packages to register and use your dependencies. If you're familiar with ASP.NET Core, the experience will be similar.
```

alina-tincas commented on .github/styles/UmbracoDocs/Acronyms.yml:
```diff
@@ -62,3 +63,72 @@ exceptions:
+  - SCM
+  - PUT
+  - POST
+  - GET
```
GET already exists
```suggestion
```

alina-tincas replied:
on line 24

alina-tincas commented on 11/umbraco-cms/reference/using-ioc.md:
```diff
@@ -4,7 +4,7 @@ description: Inversion of Control/Dependency Injection in Umbraco
 # Inversion of Control / Dependency injection
 
-Umbraco 9 supports dependency injection out of the box. Umbraco uses the [ASP.NET Core built-in dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#service-lifetimes), this means that you don't have to install an external package to register and use your dependencies, and if you're familiar with APS.NET Core, the experience will ...
+Umbraco 9 supports dependency injection out of the box. Umbraco uses the [ASP.NET Core built-in dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#service-lifetimes). This means that you don't have to install external packages to register and use your dependencies. If you're familiar with ASP.NET Core, the experience will be si ...
```
"Umbraco v9+ supports..."

liamlaverty replied:
Should be updated to "v9+" in the latest commit

alina-tincas commented on 11/umbraco-cms/tutorials/starter-kit/lessons/3-add-open-graph/step-4.md:
```diff
@@ -73,7 +73,7 @@ The final view should look like this:
 If you do not have Starter Kit installed, or experience issues with the properties not being picked up/showing up in HTML feel free to remove the fallback to siteName.
 
-If your meta properties do not show up on social media, make sure to inspect source HMTL and make sure there are no inline HTML tags in `og:title`, `og:description` etc.
+If your meta properties do not show up on social media, make sure to inspect source HTML. make sure there are no inline HTML tags in `og:title`, `og:description` etc.
```
```suggestion
If your meta properties do not show up on social media, make sure to inspect the source HTML. Make sure there are no inline HTML tags in `og:title`, `og:description` etc.
```

alina-tincas commented on 12/umbraco-cms/reference/using-ioc.md:
```diff
@@ -4,7 +4,7 @@ description: Inversion of Control/Dependency Injection in Umbraco
 # Inversion of Control / Dependency injection
 
-Umbraco 9 supports dependency injection out of the box. Umbraco uses the [ASP.NET Core built-in dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#service-lifetimes), this means that you don't have to install an external package to register and use your dependencies, and if you're familiar with APS.NET Core, the experience will ...
+Umbraco 9 supports dependency injection out of the box. Umbraco uses the [ASP.NET Core built-in dependency injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#service-lifetimes). This means that you don't have to install external packages to register and use your dependencies. If you're familiar with ASP.NET Core, the experience will be si ...
```
"Umbraco v9+ supports..."

liamlaverty replied:
Should be updated to "v9+" in the latest commit

alina-tincas commented on umbraco-cloud/getting-started/team-workflow-on-cloud.md:
```diff
@@ -2,15 +2,15 @@
 ### Create branches locally
 
-Umbraco Cloud is built on top of GIT which means that you can create branches locally as either a feature or developer branch. You can then work on the project and test out the new features before merging it into the master branch which can then be pushed up to your cloud environments.
+Umbraco Cloud is built on top of Git which means that you can create branches locally as either a feature or developer branch. You can then work on the project and test out the new features before merging it into the main branch. The branch can then be pushed up to your cloud environments.
```
The main branch we have on Umbraco Cloud is called "master" not "main". But if you prefer you can use both: "main master branch". For example if you try to clone down a cloud project, make some changes to it, then you will see that the branch you will push to is called "master". This "master branch" is also the only one we maintain and keep on Cloud (you can read more about this [here](https://docs.umbraco.com/umbraco-cloud/getting-started/git-repository-in-a-cloud-project#:~:text=We%20only%20guarantee%20to%20maintain%20and%20keep%20the%20master%20branch.))
```suggestion
Umbraco Cloud is built on top of Git which means that you can create branches locally as either a feature or developer branch. You can then work on the project and test out the new features before merging it into the master branch. The branch can then be pushed up to your cloud environments.
```

alina-tincas reviewed (changes requested):
Hi @liamlaverty, thank you for this awesome PR 🙌 

I have a few suggestions, can you take a look into them and let me know what you think? 💡 

As for the Acronyms, I will need to consult with my team about this. Can I ask in this case where did you get them from? 😊

liamlaverty commented:
> As for the Acronyms, I will need to consult with my team about this. Can I ask in this case where did you get them from?

I went through the Vale reports and just added all the common items that I recognised. I've added notes to them below, which will hopefully help understand why I've suggested them. 

- RGBA: Red/Green/Blue/Alpha colour model
   - IDE: Integrated Dev Environment (eg. Visual Studio)
   - CPU: Central Processor Unit
   - RTE: Rich Text Editor
   - JIT: Just In Time
   - MSDN: Microsoft Developer Network
   - CMD: Command keyboard shortcut
   - SMTP: Simple Mail Transfer Protocol 
   - TLS: Transport Layer Security
   - HSTS: HTTP Strict Transport Security
   - GUI: Graphical User Interface
   - RAM: Random Access Memory
   - TCP: Transmission Control Protocol
   - CNAME: CanonicalNAME (DNS record)
   - TLS: Transport Layer Security
   - TDE: Transparent Data Encryption
   - CIDR: Classless Inter-Domain Routing
   - MFA: Multi-Factor Auth
   - CDN: Content Delivery Network
   - CAA: Certificate Authority Authorisation (DNS record)
   - SVG: Scalable Vector Graphic
   - SEO: Search Engine Optimisation
   - AAAA: A DNS record
   - WAF: Web Application Firewall
   - TXT: A DNS record
   - TTL: Time To Live
   - SDK: Software Development Kit
   - SCM: Software Confg Management
   - PUT: A type of HTTP request
   - POST: A type of HTTP request
   - DELETE: A type of HTTP request
   - BCC: Blind Carbon Copy (email)
   - RCL: ???
   - MIME: Multipurpose Internet Mail Extensions (mail protocol)
   - YSOD: Yellow Screen Of Death
   - DTO: Data Transfer Object
   - ZIP: a zipped file
   - MIT: Mas. Institute of Technology (a software license)
   - CTRL: key on the keyboard
   - SPACE: key on the keyboard
   - ENTER: key on the keyboard
   - ASCII: American Standard Code for Information Interchange (character encoding standard)
   - SSL: Secure Socket Layer
   - VIP: Very Important Person
   - LINQ: Language Integrated Query
   - CRUD: Create/Read/Update/Delete
   - AKA: Also Known As
   - FIPS: Federal Info Processing Standard
   - CSV: Comma Separated Values
   - DFS: Distributed File System
   - PVC: Property Value Converter
   - POCO: Plain Old C# Object
   - SUITS: Umbraco team
   - UUI: Universally Unique ID
   - EOL: End of Life
   - LTS: Long Term Service
   - TORT: legal term
   - UTF: Unicode Transformation Format
   - CLR: Common Lang Runtiume
   - JWT: Json Web Token
   - DEBUG: application mode
   - INFO: application mode
   - TEMP: common directory
   - RFC: Request For Comments
   - GIF: Graphics Interchange Format
   - SAS: ???
   - GDPR: EU data protection law
   - CRM: Client Relation Management software
   - DAM: Digital Asset Management
   - PIM: Product Info Management

alina-tincas commented:
Thank you for elaborating @liamlaverty 🙌 

I have just discussed these acronyms with my team and below you can find the same list where we have written one of the 3 things:
1. "Already have it" - this means that this acronym already exists in the acronyms.yml file so you can remove those as the all ready exist
2. "accepted" - this means you can go ahead and add them to the file, as they do not have a definition or are self-explanatory and can be understood what they mean
3. "not accepted" means that we do not accept those acronyms as they are not self-explanatory and cannot be understood if they are not defined. Some are of course used quite often and most of us know them, but we also need to think of those who are new in IT and do not know all of these acronyms. For sure adding all those acronyms can resolve our Vale error reports, but then it defeats the purpose of making our documentation readable and understandable. So instead of removing those errors, they should be resolved. We still highly appreciate that you are trying to help us make our documentation better with this PR #H5YR

Let me know if you have any further questions on the above. 👍 

**RGBA**: Red/Green/Blue/Alpha colour model - **Already have it**
**IDE**: Integrated Dev Environment (eg. Visual Studio) - **Already have it**
**CPU**: Central Processor Unit -  **Already have it**
**RTE**: Rich Text Editor - **not accepted** 
**JIT**: Just In Time - **not accepted**
**MSDN**: Microsoft Developer Network - **not accepted**
**CMD**: Command keyboard shortcut - **not accepted** 
**SMTP**: Simple Mail Transfer Protocol - **not accepted** 
**TLS**: Transport Layer Security - **not accepted** 
**HSTS**: HTTP Strict Transport Security - **not accepted**
**GUI**: Graphical User Interface - **not accepted**
**RAM**: Random Access Memory - **accepted**
**TCP**: Transmission Control Protocol - **not accepted**
**CNAME**: CanonicalNAME (DNS record) - **accepted**
**TLS**: Transport Layer Security - **not accepted**
**TDE**: Transparent Data Encryption - **not accepted**
**CIDR**: Classless Inter-Domain Routing - **not accepted**
**MFA**: Multi-Factor Auth - **not accepted**
**CDN**: Content Delivery Network - **not accepted**
**CAA**: Certificate Authority Authorisation (DNS record) - **not accepted**
**SVG**: Scalable Vector Graphic - **not accepted**
**SEO**: Search Engine Optimisation - **not accepted**
**AAAA**: A DNS record - **accepted**
**WAF**: Web Application Firewall - **not accepted**
**TXT**: A DNS record - **accepted**
**TTL**: Time To Live - **not accepted**
**SDK**: Software Development Kit - **not accepted**
**SCM**: Software Confg Management - **not accepted**
**PUT**: A type of HTTP request - **accepted**
**POST**: A type of HTTP request - **Already have it**
**DELETE**: A type of HTTP request - **accepted**
**BCC**: Blind Carbon Copy (email) - **not accepted**
**RCL**: ??? - **not accepted**
**MIME**: Multipurpose Internet Mail Extensions (mail protocol) - **not accepted**
**YSOD**: Yellow Screen Of Death - **not accepted**
**DTO**: Data Transfer Object - **not accepted**
**ZIP**: a zipped file - **accepted**
**MIT**: Mas. Institute of Technology (a software license) - **not accepted**
**CTRL**: key on the keyboard - **accepted**
**SPACE**: key on the keyboard - **accepted**
**ENTER**: key on the keyboard - **accepted**
**ASCII**: American Standard Code for Information Interchange (character encoding standard) - **not accepted**
**SSL**: Secure Socket Layer - **not accepted**
**VIP**: Very Important Person - **accepted**
**LINQ**: Language Integrated Query - **not accepted**
**CRUD**: Create/Read/Update/Delete - **not accepted**
**AKA**: Also Known As - **not accepted** (we have just added it as a warning in the name.yml file instead)
**FIPS**: Federal Info Processing Standard - **not accepted**
**CSV**: Comma Separated Values - **not accepted**
**DFS**: Distributed File System - **not accepted**
**PVC**: Property Value Converter - **not accepted**
**POCO**: Plain Old C# Object - **not accepted**
**SUITS**: Umbraco team - **not accepted**  (we have just added it as a warning in the name.yml file instead to use sales)
**UUI**: Universally Unique ID - **not accepted**
**EOL**: End of Life - **not accepted**
**LTS**: Long Term Service - **not accepted**
**TORT**: legal term - **not accepted**
**UTF**: Unicode Transformation Format - **not accepted**
**CLR**: Common Lang Runtiume - **not accepted**
**JWT**: Json Web Token - **not accepted**
**DEBUG**: application mode - **accepted**
**INFO**: application mode - **accepted**
**TEMP**: common directory - **accepted**
**RFC**: Request For Comments - **not accepted**
**GIF**: Graphics Interchange Format - **accepted**
**SAS**: ??? - **not accepted**
**GDPR**: EU data protection law - **accepted**
**CRM**: Client Relation Management software - **not accepted**
**DAM**: Digital Asset Management - **not accepted**
**PIM**: Product Info Management - **not accepted**

liamlaverty commented:
Hey @alina-tincas , these fixes should be applied in https://github.com/umbraco/UmbracoDocs/pull/5400/commits/8cfeecf9d9a1829b383e63365abe336d1d2c669d

alina-tincas commented on 12/umbraco-cms/extending/backoffice-tours.md:
```diff
@@ -90,7 +90,7 @@ Below is an explanation of each of the properties on the tour configuration obje
     ![Content Type specific tours](<../../../10/umbraco-cms/extending/images/contentTypespecific (1) (1) (1) (1) (1).png>)
 
-    In the image above, two tours are avaibable on the _Welcome_ node:
+    In the image above, two tours are available on the _Welcome_ node:
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/extending/language-files.md:
```diff
@@ -37,7 +37,7 @@ Current languages that are included in new Umbraco installations are:
 * Turkish
 * Welsh
-* Ukranian
+* Ukrainian
```
to keep the docs consistent, can you please make this change to v11 as well?

alina-tincas commented on 12/umbraco-cms/fundamentals/backoffice/settings-dashboards.md:
```diff
@@ -30,7 +30,7 @@ For more information about Examine Management, see the [Examine Management](../.
 <summary>Published Status</summary>
 
-The Published Status dashboard displays the status of your site in the Published Cache Status section alongwith the Content and Media nodes value. The Caches section provides three options: Memory Cache, Database Cache, and Internals (NuCache).
+The Published Status dashboard displays the status of your site in the Published Cache Status section alongside the Content and Media nodes value. The Caches section provides three options: Memory Cache, Database Cache, and Internals (NuCache).
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/fundamentals/code/debugging/logging.md:
```diff
@@ -150,4 +150,4 @@ If you are interested in learning more then the following resources will benefic
 * [Nicholas Blumhardt Blog, creator of Serilog](https://nblumhardt.com/)
 * [Serilog Pluralsight Course](https://www.pluralsight.com/courses/modern-structured-logging-serilog-seq)
-* [Seq](https://getseq.net/) This is FREE for a single machine such as your own local development computer
+* [Seq](https://getseq.net/) This is **free** for a single machine such as your own local development computer
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/reference/cache/README.md:
```diff
@@ -30,11 +30,11 @@ The interface consists of the following methods:
   * This would invalidate or refresh all caches of the caching type that this `ICacheRefresher` is created for. For example, if you were caching `Employee` objects, this method would invalidate all `Employee` caches.
 * `void Refresh(int Id);`
-  * This would invalidate or refresh a single cache for an object with the provided INT id.
+  * This would invalidate or refresh a single cache for an object with the provided `int` id.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/reference/cache/README.md:
```diff
@@ -30,11 +30,11 @@ The interface consists of the following methods:
   * This would invalidate or refresh a single cache for an object with the provided GUID id.
 * `void Remove(int Id);`
-  * This would invalidate a single cache for an object with the provided INT id. In many cases Remove and Refresh perform the same operation but in some cases `Refresh` doesn't remove/invalidate a cache entry, it might update it. `Remove` is specifically used to remove/invalidate a cache entry.
+  * This would invalidate a single cache for an object with the provided `int` id. In many cases Remove and Refresh perform the same operation but in some cases `Refresh` doesn't remove/invalidate a cache entry, it might update it. `Remove` is specifically used to remove/invalidate a cache entry.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/reference/security/setup-umbraco-for-a-fips-server.md:
```diff
@@ -3,7 +3,7 @@
 
 {% hint style="warning" %}
-FIPS should only be added for compliance. It is **NOT** a recommended approach for added security. For more information read [Why Microsoft is not recommending "FIPS Mode" anymore.](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/why-we-8217-re-not-recommending-8220-fips-mode-8221-anymore/ba-p/701037)
+FIPS should only be added for compliance. It is **not** a recommended approach for added security. For more information read [Why Microsoft is not recommending "FIPS Mode" anymore.](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/why-we-8217-re-not-recommending-8220-fips-mode-8221-anymore/ba-p/701037)
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/reference/templating/macros/managing-macros.md:
```diff
@@ -74,7 +74,7 @@ This list defines the different types of macro parameters:
 * Content Type picker (`contentType`) - Choose a type from existing Document Types
 * Decimal (`Umbraco.Decimal`) - accepts only numbers
-* Email address (`Umbraco.EmailAddress`) - validates to valid email adresses
+* Email address (`Umbraco.EmailAddress`) - validates to valid email addresses
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/reference/templating/macros/managing-macros.md:
```diff
@@ -87,7 +87,7 @@ This list defines the different types of macro parameters:
 * Media picker (`Umbraco.MediaPicker`) - select a single media item
 * Content Picker (`Umbraco.ContentPicker`) - select a single content node from the Content tree
-* Form Picker (`UmbracoForms.FormsPicker`) - choose from exising Umbraco Forms
+* Form Picker (`UmbracoForms.FormsPicker`) - choose from existing Umbraco Forms
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/reference/templating/macros/partial-view-macros.md:
````diff
@@ -15,7 +15,7 @@ All Partial View Macro views inherit from `Umbraco.Cms.Web.Common.Macros.Partial
 ```
 
-The model type for a Partial View Macro is `Umbraco.Cms.Core.Models.PartialViewMacroModel` which contains all of the properties you need to render out content alongwith some additional
+The model type for a Partial View Macro is `Umbraco.Cms.Core.Models.PartialViewMacroModel`. This contains all of the properties you need to render out content alongside some additional
````
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/add-google-authentication.md:
```diff
@@ -332,7 +332,7 @@ public void ConfigureServices(IServiceCollection services)
 
 {% hint style="info" %}
-If auto-linking is disabled, the user will need to follow these steps in order to be able to use the Google Authenticaion:
+If auto-linking is disabled, the user will need to follow these steps in order to be able to use the Google Authentication:
```
to keep the docs consistent, can you please make this change to v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-a-basic-website/displaying-the-document-type-properties.md:
```diff
@@ -40,6 +40,6 @@ To set the Document Type properties:
 Reload your homepage to view the content. You should see something similar like the image below:
 
-![Displaying Documen Type Properties](../../../../10/umbraco-cms/tutorials/creating-a-basic-website/images/figure-22-displaying-document-type-properties.png)
+![Displaying Document Type Properties](../../../../10/umbraco-cms/tutorials/creating-a-basic-website/images/figure-22-displaying-document-type-properties.png)
```
to keep the docs consistent, can you please make this change to v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-a-basic-website/document-types.md:
```diff
@@ -36,7 +36,7 @@ With the help of icons, you can identify different Document Types in the **Conte
 
     ![Selecting an icon](images/figure-9-adding-an-icon-to-document-type-v11.png)
-2. Browse through the icon list and selct the icon of your choice.
+2. Browse through the icon list and select the icon of your choice.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-a-property-editor/part-2.md:
```diff
@@ -20,7 +20,7 @@ To add configuration options to our Suggestion Data Type, open the `package.mani
 "editor": {
                 "view": "~/App_Plugins/Suggestions/suggestion.html"
-            }, // Remeber a comma seperator here at the end of the editor block!
+            }, // Remember a comma separator here at the end of the editor block!
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-a-property-editor/part-2.md:
```diff
@@ -79,7 +79,7 @@ Your `package.manifest` file should now look something like this:
         /*Optional: Add 'read-only' support. Available from Umbraco 10.2+*/
         "supportsReadOnly": true
-      }, // Remeber a comma seperator here at the end of the editor block!
+      }, // Remember a comma separator here at the end of the editor block!
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-and-distributing-a-package.md:
```diff
@@ -62,7 +62,7 @@ dotnet new umbraco-v8-package -n PackageWorkshop -d
 This will create a new package called `PackageWorkshop`, and add a custom Dashboard for us to use in this tutorial. By default you will also get a Github Action added that we will return to later.
 
-Afer running you will have a folder called `PackageWorkshop`, inside that you will have your site and solution files. So try to open it in Visual Studio or Rider by opening the `PackageWorkshop.sln` file.
+After running you will have a folder called `PackageWorkshop`, inside that you will have your site and solution files. So try to open it in Visual Studio or Rider by opening the `PackageWorkshop.sln` file.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-and-distributing-a-package.md:
```diff
@@ -212,7 +212,7 @@ You don't have to worry too much about guid and orgName, they are references to
 \~/App\_Plugins/PackageWorkshop/Dashboard/main.css \~/App\_Plugins/PackageWorkshop/ContentApp/main.css
 
-Both would have the same name and be thrown in the root of the zip file - this would cause a conflict and Umbraco would rename them to randomly generated names. These names would correspond to the `guid` property, while the `orgName` and `orgPath` would make up the original names - and would be where the package would place and name the file when being intalled.
+Both would have the same name and be thrown in the root of the zip file. This would cause a conflict and Umbraco would rename them to randomly generated names. These names would correspond to the `guid` property, while the `orgName` and `orgPath` would make up the original names. These would be where the package would place and name the file when being installed.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-custom-views-for-blocklist.md:
```diff
@@ -161,7 +161,7 @@ Additionally, we need to update our stylesheet to use the color configuration. F
 
 body {
- font-family: 'Source Sans Pro', sans-seif;   
+ font-family: 'Source Sans Pro', sans-serif;   
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/creating-custom-views-for-blocklist.md:
```diff
@@ -198,7 +198,7 @@ Now that we have updated our view and stylesheet let's assign it to our block.
 2. Click the `cog` wheel next to **Features**.
 3. Select the **Product - Features - Block List**. The **Editor Settings** window opens.
-4. Select **Feature** from the **Availabe Blocks** configuration. The **Configuration of 'Feature'** window opens.
+4. Select **Feature** from the **Available Blocks** configuration. The **Configuration of 'Feature'** window opens.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/custom-error-page.md:
```diff
@@ -238,7 +238,7 @@ If you set up everything correctly and the error pages are not showing correctly
 * Custom [ContentFinders](../reference/routing/request-pipeline/icontentfinder.md) in your solution,
 * Any packages that allow you to customize redirects, or
-* Rewrite rules in web.config that might interefere with custom error handling.
+* Rewrite rules in web.config that might interfere with custom error handling.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/creating-saving-and-publishing-content.md:
```diff
@@ -51,7 +51,7 @@ To set up scheduled publishing, follow these steps:
 4.  In the **Scheduled Publishing** window, set the date and time in the **Publish at** field.
 
-    ![Schedualed publishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_publishing\_v9.png)
+    ![Scheduled publishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_publishing\_v9.png)
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/creating-saving-and-publishing-content.md:
```diff
@@ -90,7 +90,7 @@ To set up scheduled unpublishing, follow these steps:
 4.  In the **Scheduled Publishing** window, set the date and time in the **Unpublish at** field.
 
-    ![Schedualed unpublishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_Unpublishing\_v9.png)
+    ![Scheduled unpublishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_Unpublishing\_v9.png)
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/multilanguage-setup.md:
```diff
@@ -197,7 +197,7 @@ To render dictionary items in the template, replace the text with the following
 You can assign a Translator when you need a 1-1 translation of your site. For example, let's say we originally created a website in "Danish" which works from a `.dk` domain and now there is a need for an "English" site on a `.com` domain. In this case, it might be easier to copy the entire danish site and then provide access to a Translator who can then translate the site page by page.
 
-Translators are used for the translation workflow. By default, Translators have permission to **Browse** and **Update** nodes. Someone must review the translations of site pages before publishing the nodes. For more information on managing User Groups, assigning accesss or User permissions, see the [Users](../fundamentals/data/users.md) article.
+Translators are used for the translation workflow. By default, Translators have permission to **Browse** and **Update** nodes. Someone must review the translations of site pages before publishing the nodes. For more information on managing User Groups, assigning access or User permissions, see the [Users](../fundamentals/data/users.md) article.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/overview.md:
```diff
@@ -16,7 +16,7 @@ These tutorials will take you through the process of building different types of
 These tutorials will show you how you can extend the Umbraco backoffice to create a fluid and intuitive editing experience for your content editors.
 
-<table data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>Creating a custom Dashboard</strong></td><td>Go through the steps of setting up and building a custom dashboard for the Umbraco CMS backoffice.</td><td><a href="../.gitbook/assets/dashboards-image.png">dashboards-image.png</a></td></tr><tr><td><strong>Cr ...
+<table data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>Creating a custom Dashboard</strong></td><td>Go through the steps of setting up and building a custom dashboard for the Umbraco CMS backoffice.</td><td><a href="../.gitbook/assets/dashboards-image.png">dashboards-image.png</a></td></tr><tr><td><strong>Cr ...
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas commented on 12/umbraco-cms/tutorials/starter-kit/lessons/3-add-open-graph/step-4.md:
```diff
@@ -73,7 +73,7 @@ The final view should look like this:
 If you do not have Starter Kit installed, or experience issues with the properties not being picked up/showing up in HTML feel free to remove the fallback to siteName.
 
-If your meta properties do not show up on social media, make sure to inspect source HMTL and make sure there are no inline HTML tags in `og:title`, `og:description` etc.
+If your meta properties do not show up on social media, make sure to inspect source HTML. Make sure there are no inline HTML tags in `og:title`, `og:description` etc.
```
to keep the docs consistent, can you please make this change to v10 and v11 as well?

alina-tincas reviewed (changes requested):
thank you for the fixes @liamlaverty #h5YR 

I took a look again at all changes, and noticed that you have made some styleguide changes to the v12 docs but not to v10 and v11 as well, so I was wondering if you would be up for making those changes in the v10 and v11 docs as well? I added a note for which ones you need to add v11 and/or v10 (as some docs are not in v10) 😊 

Otherwise please let me know and I can merge this in and we can make the changes afterward 👍

alina-tincas commented:
Awesome job @liamlaverty, huge #h5YR for all this work 🙌 Much appreciated! 

All looks good now so am merging this in 😊
